.circleci/config/phpunit.xml

```diff
@@ -30,7 +30,7 @@
     <!-- Example BROWSERTEST_OUTPUT_DIRECTORY value: /path/to/docrootroot/sites/simpletest/browser_output -->
     <env name="BROWSERTEST_OUTPUT_DIRECTORY" value="/var/www/html/docroot/sites/simpletest"/>
     <!-- To enable deprecation testing change `weak_vendors` to `max[self]=0`. -->
-    <env name="SYMFONY_DEPRECATIONS_HELPER" value="weak_vendors"/>
+    <env name="SYMFONY_DEPRECATIONS_HELPER" value="disabled"/>
```

Comment:
What's this all about?

Reply:
Updated phpunit changes some of the variables and the way that it handles these. H&S needs to look into deprecation checking. this just isn't the way to do it.